docusaurus.config.js

```diff
@@ -235,249 +235,224 @@ const config = {
+            ],
+            to: "/main/build/modules/gov",
+          },
+          {
+            from: [
+              "/main/modules/group/01_concepts",
+              "/main/modules/group/02_state",
+              "/main/modules/group/03_messages",
+              "/main/modules/group/04_events",
+              "/main/modules/group/05_client",
+              "/main/modules/group/06_metadata",
+            ],
+            to: "/main/build/modules/auth",
+          },
+          {
+            from: [
+              "/main/modules/mint/01_concepts",
+              "/main/modules/mint/02_state",
+              "/main/modules/mint/03_begin_block",
+              "/main/modules/mint/04_params",
+              "/main/modules/mint/05_events",
+              "/main/modules/mint/06_client",
+            ],
+            to: "/main/build/modules/mint/",
+          },
+          {
+            from: [
+              "/main/modules/nft/01_concepts",
+              "/main/modules/nft/02_state",
+              "/main/modules/nft/03_messages",
+              "/main/modules/nft/04_events",
+            ],
+            to: "/main/build/modules/nft/",
+          },
+          {
+            from: [
+              "/main/modules/params/01_keeper",
+              "/main/modules/params/02_subspace",
+            ],
+            to: "/main/build/modules/params/",
+          },
+          {
+            from: [
+              "/main/modules/slashing/01_concepts",
+              "/main/modules/slashing/02_state",
+              "/main/modules/slashing/03_messages",
+              "/main/modules/slashing/04_begin_block",
+              "/main/modules/slashing/05_hooks",
+              "/main/modules/slashing/06_events",
+              "/main/modules/slashing/07_tombstone",
+              "/main/modules/slashing/08_params",
+              "/main/modules/slashing/09_client",
+            ],
+            to: "/main/build/modules/slashing/",
+          },
+          {
+            from: [
+              "/main/modules/staking/01_state",
+              "/main/modules/staking/02_state_transitions",
+              "/main/modules/staking/03_messages",
+              "/main/modules/staking/04_begin_block",
+              "/main/modules/staking/05_end_block",
+              "/main/modules/staking/06_hooks",
+              "/main/modules/staking/07_events",
+              "/main/modules/staking/08_params",
+              "/main/modules/staking/09_client",
+            ],
+            to: "/main/build/modules/staking/",
+          },
+          {
+            from: [
+              "/main/modules/upgrade/01_concepts",
+              "/main/modules/upgrade/02_state",
+              "/main/modules/upgrade/03_events",
+              "/main/modules/upgrade/04_client",
+            ],
+            to: "/main/build/modules/upgrade/",
+          },
+          {
+            from: ["/main/modules/capability", "/main/ecosystem"],
+            to: "/main/build/modules/auth/",
```

Comment:
Imho, this should point to: https://cosmos.github.io/cosmos-sdk-docs/main/modules
Btw, this page does not exist in v0.50 docs for some reason.

Reply:
I took it out this page a while back when doing the mocks just because. its not the cleanest and overwhelming for the user

Reply:
This explains what happened with capability and is the previous ecosystem page (which links cosmwasm and Ethermint), so redirecting to with instead makes less sense.